boards/common/esp8266/board_common.c

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2018 Gunar Schorcht
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    boards_common_esp8266 ESP8266 Common
+ * @ingroup     boards_common
+ * @brief       Common files for the esp8266 board.
+ * @{
+ *
+ * @file
+ * @brief       Definitions for all esp8266 board.
+ *
+ * @author      Gunar Schorcht <gunar@schorcht.net>
+ */
+
+#include "espressif/eagle_soc.h"
+#include "board_common.h"
+#include "periph/gpio.h"
+#include "periph/spi.h"
+
+#ifdef __cplusplus
+ extern "C" {
+#endif
+
+static uint32_t leds_initialized = 0x0;
+
+void led_on_off (uint8_t led, uint8_t value)
```

Comment:
I'm not very happy that there's now ```led_*()``` in the global namespace. Maybe we can leave it for now, but expect this to clash at some point...

Reply:
Would it be ok to have a prefix ```esp8266_```. I'm not so happy to define macros with implementation details in a header file like:
```
GPIO_REG_WRITE(GPIO_OUT_W1TS_ADDRESS, BIT(led));
```